### ISSUE TITLE
Sync have stopped on block 984006

### ISSUE THREAD

**Issue by atomlab** [blockwatch-cc/tzindex#29]
I have two tzindex nodes on different servers and they are both have stopped on 984006 block. tezos-node works fine. Could you help how to fix this?

tzindex release 6.0.3
tezos-node 7.1

tzindex logs
```
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.778369 DEBG JRPC GET /chains/main/blocks/984009 HTTP/1.1
Jun 06 09:38:36 xtz-01 tzindex[6300]: Host: 127.0.0.1:8732
Jun 06 09:38:36 xtz-01 tzindex[6300]: Accept: application/json
Jun 06 09:38:36 xtz-01 tzindex[6300]: Content-Type: application/json
Jun 06 09:38:36 xtz-01 tzindex[6300]: User-Agent: Blockwatch-tzindex/dev.none
Jun 06 09:38:36 xtz-01 tzindex[6300]: [1B blob data]
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.895265 DEBG BLOC Loaded 4154 total delegates
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.895709 ERRO SRVR BW-bab46c4e920990403958e83faf1840dffe39b8dc 404 (1305) GET /explorer/account/tz1esvRfFFB6jDbmApp5t9Mb6rP73XvipK1Z/op?offset=0&limit=0 HTTP/1.0 - ReadAccountOps failed (no such account): account not indexed
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.896386 INFO BLOC Stopping blockchain sync due to too many errors at 984006.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.896395 DEBG BLOC Flushing account.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.907551 DEBG BLOC Storing account idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.910161 DEBG BLOC Flushing contract.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.912565 DEBG BLOC Storing contract idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.915035 DEBG BLOC Flushing block.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.920783 DEBG BLOC Storing block idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.923311 DEBG BLOC Flushing op.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.937975 DEBG BLOC Storing op idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.940282 DEBG BLOC Flushing flow.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.943784 DEBG BLOC Storing flow idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.946027 DEBG BLOC Flushing chain.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.948318 DEBG BLOC Storing chain idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.950939 DEBG BLOC Flushing supply.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.953617 DEBG BLOC Storing supply idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.956050 DEBG BLOC Flushing rights.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.962377 DEBG BLOC Storing rights idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.964628 DEBG BLOC Flushing snapshot.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.974686 DEBG BLOC Storing snapshot idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.977060 DEBG BLOC Flushing income.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.979568 DEBG BLOC Storing income idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.981934 DEBG BLOC Flushing election.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.984376 DEBG BLOC Flushing proposal.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.986564 DEBG BLOC Flushing vote.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.989124 DEBG BLOC Flushing ballot.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.991410 DEBG BLOC Storing gov idx tip.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.993945 DEBG BLOC Flushing bigmap.
Jun 06 09:38:36 xtz-01 tzindex[6300]: 2020/06/06 09:38:36.996545 DEBG BLOC Storing bigmap idx tip.
Jun 06 09:38:37 xtz-01 tzindex[6300]: 2020/06/06 09:38:37.001810 ERRO BLOC Exiting monitor loop on cancelled context
Jun 06 09:38:37 xtz-01 tzindex[6300]: 2020/06/06 09:38:37.001847 DEBG BLOC Block 984009 download failed: Get http://127.0.0.1:8732/chains/main/blocks/984009: context canceled
Jun 06 09:38:37 xtz-01 tzindex[6300]: 2020/06/06 09:38:37.001854 INFO BLOC Context cancelled. Stopping blockchain ingest.
Jun 06 09:38:38 xtz-01 tzindex[6300]: 2020/06/06 09:38:38.164174 ERRO SRVR BW-dc1ebf5483d37d226f4590421ebaed548af2f5f5 404 (1305) GET /explorer/account/tz1RRTiLzuWgnjSgfKSUU28PDzAPaPfsmPTx/op?offset=0&limit=0 HTTP/1.0 - ReadAccountOps failed (no such account): account not indexed
Jun 06 09:38:39 xtz-01 tzindex[6300]: 2020/06/06 09:38:39.080296 ERRO SRVR BW-f87fa7a1745f5e1e8cdec46c13f70d135c50392b 404 (1305) GET /explorer/account/tz1ivcia9HyAfoLFMN9ynKoMLCDxckbn8Qz9/op?offset=0&limit=0 HTTP/1.0 - ReadAccountOps failed (no such account): account not indexed
Jun 06 09:38:43 xtz-01 tzindex[6300]: 2020/06/06 09:38:43.543129 ERRO SRVR BW-d62a0bdb37b401e63a710ae05b0534657aa37fb6 404 (1305) GET /explorer/account/tz1UQSKz5ffeqjWZVR7Dn6GpTCytFxHRvVmx/op?offset=0&limit=0 HTTP/1.0 - ReadAccountOps failed (no such account): account not indexed
Jun 06 09:38:43 xtz-01 tzindex[6300]: 2020/06/06 09:38:43.594102 ERRO SRVR BW-c1db059e934a57ef7079e2dc60dbfbc2c22dbc69 404 (1305) GET /explorer/account/tz1XaMygPHfcvo4AAibrC41W7NZZBkF1dpku/op?offset=0&limit=0 HTTP/1.0 - ReadAccountOps failed (no such account): account not indexed
Jun 06 09:38:43 xtz-01 tzindex[6300]: 2020/06/06 09:38:43.659582 ERRO SRVR BW-299a29333570f55f683f2bdb873e2d90e8767d74 404 (1305) GET /explorer/account/tz1UqJX3NoGDZ8NDRZNTzxT2tKCvnfcuDPib/op?offset=0&limit=0 HTTP/1.0 - ReadAccountOps failed (no such account): account not indexed
Jun 06 09:38:43 xtz-01 tzindex[6300]: 2020/06/06 09:38:43.947696 ERRO SRVR BW-7e9f7d923999dce01138e4a676f5f019d53102e2 404 (1305) GET /explorer/account/tz1UYbgo2XmJ3S3BvjFH9KMNVQvwQFVvtD34/op?offset=0&limit=0 HTTP/1.0 - ReadAccountOps failed (no such account): account not indexed
Jun 06 09:38:48 xtz-01 tzindex[6300]: 2020/06/06 09:38:48.288991 DEBG BLOC Ranks built in 693.878679ms
Jun 06 09:38:50 xtz-01 tzindex[6300]: 2020/06/06 09:38:50.709765 ERRO SRVR BW-e2c15aa9bfd8cd8270665c445466b5d0017f2001 404 (1305) GET /explorer/account/tz1Zxu75fPS5vE9QdcCSBxYPUabJSCHdfmRJ/op?offset=0&limit=0 HTTP/1.0 - ReadAccountOps failed (no such account): account not indexed
```

**Comment by echa** [comment]
This has been fixed on master in [95156a8](https://github.com/blockwatch-cc/tzindex/commit/95156a8a06d5439270b2cac79e7098b48b4ec0fa).

**Comment by atomlab** [comment]
thanks, it helped me
